codesystems/CodeSystem-England-FGMRemovalReason.xml

```diff
@@ -0,0 +1,36 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-FGMRemovalReason" />
+  <url value="https://fhir.nhs.uk/CodeSystem/England-FGMRemovalReason" />
+  <identifier>
+    <system value="https://fhir.nhs.uk/identifier/CodeSystem" />
+    <value value="fgm-removal-reason" />
+  </identifier>
+  <version value="1.0.0" />
+  <name value="EnglandFGMRemovalReason" />
+  <title value="England FGM Removal Reason" />
+  <status value="draft" />
+  <experimental value="true" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A codesystem used for the removal of FGM Flag" />
```

Comment:
Need to add a < purpose > - this can use same text as < description > if that's appropriate, or you can add extra info if needed.